<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>589</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>802</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="tabPosition">
          <enum>QTabWidget::West</enum>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="DataConversionButon">
          <attribute name="title">
           <string>数据转换</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="1">
            <widget class="QPushButton" name="OutputFilsePathButton">
             <property name="text">
              <string>保存文件路径</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QTableView" name="OutPutTableView"/>
           </item>
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="FreshDataButton">
               <property name="text">
                <string>刷新数据</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="ChooseFileButon">
               <property name="text">
                <string>选择文件</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="0" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="DailyDataButton">
               <property name="text">
                <string>生成日报</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="WeeklyDataButton">
               <property name="text">
                <string>生成周报</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="MonthDataButton">
               <property name="text">
                <string>生成月报</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="SendMail">
               <property name="text">
                <string>一键发送邮件</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="MailSet">
          <attribute name="title">
           <string>邮箱设定</string>
          </attribute>
          <widget class="QWidget" name="formLayoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>641</width>
             <height>421</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>协议类型</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="ProtocolTypeBox">
              <item>
               <property name="text">
                <string>IMAP</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>POP3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Exchange</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>邮件账户</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="EmailAccountID"/>
            </item>
            <item row="9" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>发送服务器</string>
              </property>
             </widget>
            </item>
            <item row="9" column="1">
             <widget class="QLineEdit" name="SendMailServer"/>
            </item>
            <item row="10" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>端口</string>
              </property>
             </widget>
            </item>
            <item row="10" column="1">
             <widget class="QLineEdit" name="SendMailPort">
              <property name="text">
               <string>24</string>
              </property>
             </widget>
            </item>
            <item row="11" column="1">
             <widget class="QComboBox" name="SendProtocolTypeBox">
              <item>
               <property name="text">
                <string>无</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>SSL/TLS</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>STARTTLS</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="12" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>账户</string>
              </property>
             </widget>
            </item>
            <item row="12" column="1">
             <widget class="QLineEdit" name="SendMailAccount"/>
            </item>
            <item row="13" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>密码</string>
              </property>
             </widget>
            </item>
            <item row="13" column="1">
             <widget class="QLineEdit" name="SendMailPassWord"/>
            </item>
            <item row="11" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>加密</string>
              </property>
             </widget>
            </item>
            <item row="14" column="1">
             <widget class="QPushButton" name="EmailSaveButton">
              <property name="text">
               <string>保存</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="EmailPassWord"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>密码</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="TakeEmailServer"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>接收服务器</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>端口</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="Port">
              <property name="text">
               <string>143</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>加密</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QComboBox" name="EncryptionButton">
              <item>
               <property name="text">
                <string>无</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>SSL/TLS</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>STARTTLS</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>887</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
